src/main/java/org/folio/circulation/domain/policy/FixedDueDateSchedules.java

```diff
@@ -47,13 +48,21 @@ static FixedDueDateSchedules from(JsonObject representation) {
 
   private Predicate<? super JsonObject> isWithin(DateTime date) {
     return schedule -> {
-      DateTime from = DateTime.parse(schedule.getString("from"));
-      DateTime to = DateTime.parse(schedule.getString("to"));
+      LocalDate from = DateTime.parse(schedule.getString("from")).toLocalDate();
```

Comment:
Why have these been converted to local dates?

Reply:
This is due to the fact that the date in the FixedDueDateSchedules is presented in the format "2019-06-14T00:00:00.000Z" (without time) and if we set the range in the schedule from the 1st to the 30th number then this range includes the beginning of the 30th and not the full day.
But the incoming date (dueDate, loanDate) comes with time. That is, not strict equality at the end of the schedule.
If we use a LocalDate then it makes sense to compare only dates.

Reply:
Rechecked, made a mistake. The last date in the schedule contains the time of the end of the day.
Yes indeed, the application of the LocalDate is redundant.

Reply:
Revert

Reply:
You are correct that there is/was a long standing mismatch between the start and end date of fixed due date schedules, in that they should either be explicitly a date only or made to be timezone aware date times with appropriate times.

That conversation has been deferred, and is likely separate to this change